ERCS/erc-7399.md

```diff
@@ -34,7 +34,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Under this standard a flash loan is a loan of an `amount` of an [ERC-20](./eip-20.md) `asset` from a `lender`. This loan can remain open for the span of a single `flash` call in the `lender`.
 
-This `amount` plus a `fee` defined by the `lender` in the same `asset` must be repaid before the end of `flash` call at a `repayment receiver` address defined by the `lender`.
+This `amount` plus a `fee` defined by the `lender` in the same `asset` must be repaid before the end of `flash` call at a _repayment receiver_ address defined by the `lender`.
```

Comment:
I've made the change once, but for all other instances: backticks (\`) should only be used for inline code snippets. "repayment receiver" and "lender" don't seem to be declared identifiers, so you should use another formatting to indicate their significance (like italics maybe).